ci/pipeline.yml

```diff
@@ -126,6 +128,84 @@ jobs:
         status: failure
         context: unit-tests
 
+  - name: shipit
+    public: true
+    serial: true
+    plan:
+    - do:
+      - in_parallel:
+          - { get: git,     passed: [rc] }
+      - task: release
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: cf-routing.common.repositories.cloud.sap/pcap-release-testflight
+              tag:        semantic
```

Comment:
Will this also still change?
Eventually we could have this on `latest` or some other commonly used tag name.

Reply:
yes, we will tag this as "latest" once we are sure we didn't break anything

Reply:
image tagged to latest.